ui/components/app/create-new-vault/create-new-vault.js

```diff
@@ -144,7 +143,7 @@ export default function CreateNewVault({
       <Button
         data-testid="create-new-vault-submit-button"
         className="create-new-vault__submit-button"
-        type="primary"
+        variant={BUTTON_VARIANT.PRIMARY}
         submit
         disabled={!isValid}
       >
```

Comment:
Hey @PrgrmrHarshShukla, there are some props here that aren't available in the `component-library` version like `submit`. Could you double-check the props and adjust accordingly?

Reply:
Sure @georgewrmarshall ,
Will do it and report back.

Thanks.

Reply:
Hey @georgewrmarshall ,
I could not find the corresponding prop for `submit` in `component-library`, if there is one.
Should I just remove it?

Please help.

Reply:
After looking at the old button component I think it can be removed 
https://github.com/MetaMask/metamask-extension/blob/6125838566e0096ce6f2e4f6c90eeb0f47defca4/ui/components/ui/button/button.component.js#L53

Reply:
Hey @georgewrmarshall ,
👍Done.
Please look into this.